gnome-inform7.spec

```diff
@@ -49,6 +54,11 @@ make
 %install
 rm -rf %{buildroot}
 make install DESTDIR=%{buildroot}
+mkdir "%{buildroot}/usr/local"
+mkdir "%{buildroot}/usr/local/lib"
```

Comment:
Handy tip, you can do these both at once with `mkdir -p "%{buildroot}/usr/local/lib"`

That said, it may be possible just to stick these in `%{_libdir}`, instead, and not have to create the directory. If it's possible, I think that would be better. Since Fedora doesn't have these libraries it's impossible they would conflict with another package, but `/usr/local` is normally reserved for users to put their own stuff into so it could theoretically conflict with some user's stuff.

Reply:
I moved the libraries to %{_libdir} and tested on Fedora29, everything looks good.  Pushed the change.  Will test on Fedora30 too and let you know the result.